specifyweb/frontend/js_src/lib/components/Toolbar/WbsDialog.tsx

```diff
@@ -129,159 +133,246 @@ function TableHeader({
   );
 }
 
-/** Render a dialog for choosing a data set */
-export function DataSetsDialog({
+
+type WB_VARIANT = keyof Omit<typeof datasetVariants, "bulkAttachment">;
```

Comment:
```suggestion
type WbVariant = keyof Omit<typeof datasetVariants, "bulkAttachment">;
```

to be consistent with our naming